src/Microsoft.Diagnostics.Monitoring.WebApi/Models/PackageMode.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Models
+{
+    [Flags]
+    public enum PackageMode
```

Comment:
I'm guessing the intent is that the upper WORD is the "what to include" and the lower WORD is the package format and that we want to maintain this separation so that we can easily add more of each type in the future? If so, can you add comments to this effect?